sdks/python/apache_beam/runners/common.py

```diff
@@ -789,7 +789,7 @@ def finalize(self):
 
   def _reraise_augmented(self, exn):
     if getattr(exn, '_tagged_with_step', False) or not self.step_name:
-      raise
+      raise exn
```

Comment:
Would not this override the stack trace? I believe it should be either raise or raise exn[0], exn[1], exn[2] format.

Reply:
Ooops. Good catch.